src/spikeinterface/extractors/cellexplorersortingextractor.py

```diff
@@ -149,15 +150,29 @@ def __init__(
         sorting_segment = CellExplorerSortingSegment(spiketrains_dict, unit_ids)
         self.add_sorting_segment(sorting_segment)
 
-        self.extra_requirements.append("scipy")
+        self.extra_requirements.append(["scipy", "pymatreader"])
```

Comment:
Do we still need `scipy` then?

Reply:
This other library usew scipy under the hood, where do we use this extra requirements btw?

Reply:
In the containers, to install all requirements to reload the extractor ;) If `scipy` is a requirement of `pymatreader`, then we can drop it!

Reply:
Thanks, let's drop it.

Reply:
Removed.